generators/client/templates/angular/src/main/webapp/app/admin/configuration/_configuration.service.ts

```diff
@@ -27,7 +30,7 @@ export class <%=jhiPrefixCapitalized%>ConfigurationService {
     }
 
     get(): Observable<any> {
-        return this.http.get('management/configprops').map((res: Response) => {
+        return this.http.get(<%- apiUrlPrefix %>management/configprops').map((res: Response) => {
```

Comment:
As @ruddel highlighted, sorry i didn't catch it earlier, not all services are handled by uaa, hence this will not work. Only auth related services need uaa prefix others can always have just the server prefix

Reply:
@deepu105 understood, what I don't get is why does the audits service requires the uaa prefix because they are not auth related as I understand.
And if I understand you correctly, the server prefix must be added for entity services, etc. as well when using uaa? This is currently not the case.
```
@Injectable()
export class FieldTestEntityService {

    private resourceUrl = 'api/field-test-entities';

    constructor(private http: Http, private dateUtils: JhiDateUtils) { }
`
``

Reply:
- for audits I'm not sure this even makes sense, as if you use UAA, then the audit page shouldn't be available
- for entities this might be an error, as I didn't test the SERVER_API_URL with UAA, but I don't understand why this would need this prefix

Reply:
@jdubois 
* the sample `app-ng2-gateway-uaa` includes the audits page
* ok, then I don't use it for uua
Will update the PR soon.

Reply:
@xetys can you confirm this